src/content/posts/how-to-raise-funds/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: How to Raise Funds Using Angor
+published: 2024-06-17
+description: 'Angor is a decentralized crowdfunding platform on Bitcoin, offering secure fund management and direct investor engagement for transparent project funding.'
+image: './raise.svg'
+tags:  [Bitcoin, Technology, Education]
+category: 'Angor'
+draft: false 
+---
+### Raise Funds Securely with Angor:  Step-by-Step Guide
+
+Angor is a decentralized crowdfunding platform built on Bitcoin and uses Nostr for enhanced security and transparency. It allows founders to raise funds for their projects while maintaining control over the funds and fostering direct communication with investors. Here’s a step-by-step guide on how to raise funds using Angor.
+
+### Create a Wallet
+
+Before you can start raising funds, you need to set up a digital wallet on Angor. If you don’t have a wallet yet, create a wallet to use Angor.
+
+- **Navigate to the Wallet Section**: Once logged in, find and click on the 'Wallet' section from your dashboard.
+
+- **Create Your Wallet**: Follow the on-screen instructions to create your digital wallet. This typically involves:
+
+    - Sign up on the Angor platform.
+    - Navigate to the wallet creation section.
+    - Click on "Create Wallet."
+    - Angor will automatically set up the wallet for you.
+    - Set a strong password to protect your wallet.
+
+        ![Wallet after creation](./assets/wallet-after-creation.svg)
+
+- **Back Up Recovery Phrases**: You’ll be given a set of recovery phrases. Store these securely, as they are essential for accessing your wallet if you forget your password.
+
+Taking the time to thoroughly explore the projects will help you make an informed decision about where to invest and how to raise your funds.
+
+### Create a Project
+
+Once your wallet is set up, you can create a project to start raising funds. Here are the steps involved, illustrated with images of the Angor platform interface.
+
+### Step 1: Fill in Project Metadata
+
+- **Navigate to the "Create Project" Section**: Navigate to the 'Founder' section and click on 'Create Project'.
+
+- **Enter Project Metadata**:
+  - **Project Name**: Enter a clear and descriptive name for your project.
+  - **About**: Provide detailed information about your project, including its objectives and vision.
+  - **Project Website**: Enter the URL of your project’s website, if available.
+  - **Banner**: Enter the URL of your project’s banner image to make your project page visually appealing.
+  - **Nip 05 and Nip 57 (zaps)**: Fill these fields with the relevant information if applicable.
```

Comment:
I think we should link here to explain what zaps are?

Reply:
Adding info about zaps would be great